build/build.go

```diff
@@ -37,7 +37,7 @@ func main() {
 	}
 
 	timeStr := time.Now().UTC().Format("20060102150405")
-	ldFlags := fmt.Sprintf("-X bosun.org/version.VersionSHA %s -X bosun.org/version.VersionDate %s", sha, timeStr)
```

Comment:
What error is this fixing? This code successfully sets the appropriate variables in all of our builds. Is it not working for you?